### PR TITLE
Typography and list

### PR DESCRIPTION
**Pull request by amirnathani9**
## change logs
-added Typography
  -heading
  -font size
  -gray text
  -text alignment

-added List
  - basic list
  - list with avatar
  - list with icon
  - list with check box

## Gifs

typography
list

## Preview 
[Typography Preview](https://deploy-preview-5--nomark-ui.netlify.app/components/typography/typography.html)
[List Preview](https://deploy-preview-5--nomark-ui.netlify.app/components/list/list.html)

### CODE REVIEW

**Review by devdeadviz — APPROVED** [review]
Looks good to me. Good work buddy!

File: styles/variable.css [hunk 1]
```diff
@@ -14,6 +14,8 @@
     --danger-dark-color: #85121b;
     --warning-color: #fff3cd;
     --warning-dark-color: #947005;
+    --gray-dark-color:#757575;   
     --transparent-color-1:rgba(2, 129, 149, 0.5);
     --transparent-color-2: rgba(0, 0, 0, 0.5);
```
Comment (bishnoimukesh) [inline styles/variable.css:19]: use hex code throughout

**Review by bishnoimukesh — COMMENTED** [review]
looks good to merge with single change

File: styles/variable.css (hunk 1 above)
Comment (amirnathani9) [reply styles/variable.css:19]: here using because to give the opacity and make it transparent

**Pull request merged**